app/views/miq_action/edit.html.haml

```diff
@@ -1 +1,11 @@
 = render :partial => 'form'
+= react('ActionForm', 
+         :recordId => @action.id.to_s,
+         :inheritTags => @edit[:cats],
+         :availableAlerts => @edit[:allChoices].sort,
```

Comment:
why is this one `@edit[:allChoices].sort` while the similar one in `app/views/miq_action/new.html.haml` is `@edit[:choices].sort`?